build/BUILD.dawn

```diff
@@ -1020,6 +1028,8 @@ cc_library(
         "src/dawn/wire/client/ClientInlineMemoryTransferService.cpp",
         "src/dawn/wire/client/Device.cpp",
         "src/dawn/wire/client/Device.h",
+        "src/dawn/wire/client/EventManager.cpp",
```

Comment:
Looked into the build file a bit more – `dawn_wire` (and by extension `generate_wire`) is currently unused right? Feel free to keep it if we plan to use it in the future.

Reply:
Yes, they are unused at the moment. However, the plan is to have an option of using an external process to access the GPU, and when that happens this wire protocol is what will be used to communicate between these two processes.

I have began work on this but the various CODE * initiatives have taken priority...